keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/docs/Literal.kt

```diff
@@ -0,0 +1,7 @@
+package com.netflix.spinnaker.keel.api.docs
+
+import kotlin.annotation.AnnotationTarget.CLASS
+
+@Target(CLASS)
+// TODO: not a great name
+annotation class Literal(val type: String = "string", val value: String)
```

Comment:
Alias for the way we use Swagger's `@Schema` on `AllPorts`

Reply:
`@Constant`?

Reply:
I think `@Literal` is fine.